packages/app/src/systems/Settings/pages/ViewSeedPhrase/ViewSeedPhrase.tsx

```diff
@@ -37,7 +37,7 @@ export function ViewRecoveryPhrase() {
           <Alert status="warning" css={styles.alert}>
             <Alert.Description>
               <Text fontSize="xs" css={styles.alertDescription}>
-                DON&APOS;T SHARE your Recovery Phrase. {'\n'}
+                DON&APOS;T SHARE your Seed Phrase. {'\n'}
```

Comment:
```suggestion
                DON&apos;T SHARE your Seed Phrase. {'\n'}
```

<img width="373" alt="Screenshot 2023-05-18 at 7 10 50 PM" src="https://github.com/FuelLabs/fuels-wallet/assets/15849997/0d383244-b1e4-43dd-9835-eaf951df45fd">